packages/app-js/src/Editor.tsx

```diff
@@ -64,10 +55,10 @@ export default class Editor extends React.PureComponent<Props> {
 
   componentDidUpdate () {
     const { code, onEdit, snippet } = this.props;
-
```

Comment:
The extra break between the var defs and the code block here, doesn't cost anything :)

Reply:
added.